x-pack/test/upgrade_assistant_integration/upgrade_assistant/reindexing.js

```diff
@@ -83,6 +83,29 @@ export default function ({ getService }) {
       });
     });
 
+    it('can resume after reindexing was stopped right after creating the new index', async () => {
```

Comment:
So these are API integration tests in a dedicated `upgrade_assistant_integration` folder?

Reply:
Yeah, the api integration tests for reindexing live in a separate folder. Not sure if its because of the specific config changes it has that diverge with the one specified in `api_integration` (see: [this line](https://github.com/elastic/kibana/blob/7.17/x-pack/test/upgrade_assistant_integration/config.js#L35))or if it was just neglected and needs to be moved over.

Reply:
These tests existed pre-8.0 upgrade. It does have specific config changes as @sabarasaba pointed out. At one point, I wanted to look into consolidating the UA tests, as it's a bit confusing that we also have UA tests under https://github.com/elastic/kibana/tree/main/x-pack/test/api_integration/apis/upgrade_assistant 🙁 .